src/Components/AddressForm.tsx

```diff
@@ -123,7 +123,7 @@ export const AddressForm: React.FC<AddressFormProps> = ({
           placeholder="Full name"
           title={billing ? "Name on card" : "Full name"}
           autoCorrect="off"
-          value={address.name}
+          value={value?.name}
```

Comment:
This change is what allows the address form to be updated when the user selects a suggested address after verification. Before this change, the address form was only updated from user input. After this change, the parent component (`ShippingRoute`) can also tell the address form which values to show.